static/app/components/nav/tour/tour.tsx

```diff
@@ -0,0 +1,247 @@
+  StackedNavigationTour.ISSUES,
+  StackedNavigationTour.EXPLORE,
+  StackedNavigationTour.DASHBOARDS,
+  StackedNavigationTour.INSIGHTS,
+  StackedNavigationTour.SETTINGS,
+];
+
+export const STACKED_NAVIGATION_TOUR_CONTENT = {
+  [StackedNavigationTour.ISSUES]: {
+    description: t(
+      'Issues are problems detected by Sentry. Code breaks - we tell you where, when, and why.'
+    ),
+    title: t('See what broke'),
+  },
+  [StackedNavigationTour.EXPLORE]: {
+    description: t(
+      'Create queries, investigate data exemplars, and save charts for dashboards. Explore is where you turn raw data into answers.'
+    ),
+    title: t('Dig into data'),
+  },
+  [StackedNavigationTour.DASHBOARDS]: {
+    description: t(
+      'Dashboards enable you to display key insights all on one page. Create new or find existing custom dashboards here.'
+    ),
+    title: t('Track what matters'),
+  },
+  [StackedNavigationTour.INSIGHTS]: {
+    description: t(
+      'Project and domain insights live here, giving you prebuilt looks into your app’s health. '
+    ),
+    title: t('Know what’s happening'),
+  },
+  [StackedNavigationTour.SETTINGS]: {
+    description: t(
+      'Stats & Usage is now under Organization Settings. We’ve consolidated this page so you can see all your configurations in one place.'
+    ),
+    title: t('Find more in Settings'),
+  },
+};
+
+export const STACKED_NAVIGATION_TOUR_GUIDE_KEY = 'tour.stacked_navigation';
+
+export const StackedNavigationTourContext =
+  createContext<TourContextType<StackedNavigationTour> | null>(null);
+
+export function useStackedNavigationTour(): TourContextType<StackedNavigationTour> {
+  const tourContext = useContext(StackedNavigationTourContext);
+  if (!tourContext) {
+    throw new Error('Must be used within a TourContextProvider<StackedNavigationTour>');
+  }
+  return tourContext;
+}
+
+export function NavTourElement({
+  children,
+  ...props
+}: Omit<TourElementProps<StackedNavigationTour>, 'tourContext'>) {
+  return (
+    <TourElement<StackedNavigationTour>
+      tourContext={StackedNavigationTourContext}
+      position="right-start"
+      {...props}
+    >
+      {children}
+    </TourElement>
+  );
+}
+
+function useStackedNavigationTourCompleted() {
+  const {data: assistantData} = useAssistant();
+
+  return useMemo(() => {
+    const stackedNavigationTourData = assistantData?.find(
+      item => item.guide === STACKED_NAVIGATION_TOUR_GUIDE_KEY
+    );
+
+    return stackedNavigationTourData?.seen ?? true;
+  }, [assistantData]);
+}
+
+export function NavigationTourProvider({children}: {children: React.ReactNode}) {
+  const organization = useOrganization();
+  const isStackedNavigationTourCompleted = useStackedNavigationTourCompleted();
+  const initialUrlRef = useRef<string | null>(null);
+  const navigate = useNavigate();
+  const {activeGroup, setShowTourReminder} = useNavContext();
+  const location = useLocation();
+
+  const onStartTour = useCallback(() => {
```

Comment:
There is a bit of extra work that needs to be done when starting the nav tour:

1. We need to navigate to each section when the step changes
2. We need to save the original URL and navigate back when the tour ends
3. We need to lock scrolling, because the tour CSS means that we can't use `position: sticky` or z-index while it is in progress which means you can scroll past the nav